docs/guide/java/entities.md

```diff
@@ -123,4 +121,90 @@ TODO more drivers such as jmx, etc are planned
 Testing
 -------
 
-* Run in a mock ``SimulatedLocation``, defining new metaclass methods to be able to start there and assert the correct behaviour when that is invoked
+* Unit tests can make use of `SimulatedLocation` and `TestEntity`, and can extend `BrooklynAppUnitTestSupport`.
+* Integration tests and use a `LocalhostMachineProvisioningLocation`, and can also extend `BrooklynAppUnitTestSupport`.
+
+
+<a name="SoftwareProcess-lifecycle"></a>
```

Comment:
links are now generated automatically by the doc tools, so these `<a ...` links can be removed i think.  you can even have menu-updates-as-you-scroll behaviour; see e.g. learn more -> features.